funds/privacy_guides/privacy_guides_funds.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: page
+title: "MAGIC Privacy Guides Fund"
+---
+
+*Updated 2024-03-01*
+
+## Background
+
+Multidisciplinary Academic Grants in Cryptocurrencies (**MAGIC Grants**) is a 501(c)(3) nonprofit organization. It can conduct a variety of qualifying charitable activities, which notably include the following:
+
+* Creating educational materials
+* Purchasing and distributing educational materials
+* Supporting the “maintenance of public infrastructure”
+    * “Public infrastructure” in this case refers to major public cryptocurrency networks
+    * “Supporting” refers to research, development, node operation, and security testing, among others
+
+[Privacy Guides](https://privacyguides.org) is a socially motivated website that provides information for protecting your data security and privacy (the **Project**). Their mission is to inform the public about the value of digital privacy, and global government initiatives which aim to monitor your online activity. Their website is free of advertisements and not affiliated with any of the listed providers.
+
+## Creation of the Fund
+
+At the time of the fund's creation, the five Privacy Guides team members are jonah, niek-de-wilde, dngray, freedy, and olivia.
+
+These five individuals will join as the initial five committee members.
+
+The existing assets for the Privacy Guides Project, including assets held by the Open Collective Foundation and the assets held by individual team members, will be transferred to MAGIC Grants in a restricted donation to the MAGIC Privacy Guides Fund (the **MAGIC Fund**).
```

Comment:
What will this restricted donation letter look like? Or will I have to write this? Not necessarily a problem, but I suppose I'd probably want to find a lawyer?

Reply:
You don't need to write this. I'll send you our standard donation letter. If you want to get a lawyer to confirm that the restrictions are appropriate, then you can do that.